components/event-sources/reconciler/httpsource/reconcile.go

```diff
@@ -144,6 +158,33 @@ func (r *Reconciler) reconcileSink(src *sourcesv1alpha1.HTTPSource) (*messagingv
 	return currentCh, nil
 }
 
+// reconcilePolicy reconciles the state of the Policy.
+func (r *Reconciler) reconcilePolicy(src *sourcesv1alpha1.HTTPSource, ksvc *servingv1alpha1.Service) (*authenticationv1alpha1.Policy, error) {
+	if ksvc == nil {
+		r.event(src, failedCreateReason, "Skipping creation of Istio Policy as there is no ksvc yet")
```

Comment:
It's not a failure if you're just waiting for a dependency which you expect to become ready in the future. A log at the "info" level would be more appropriate.

Reply:
Changed to a Info log